config.sample.ini

```diff
@@ -106,6 +107,7 @@ TLS = true
 SSL = false
 Sender = max.mustermann@gmail.com
 Recipients = max.mustermann@gmail.com
+; RecipientsPerItem = {"123" : ["max.mustermann@example.com", "moritz.mustermann@example.com"], "456" : ["max.mustermann@example.com"]}
```

Comment:
What about allowing a single recipient to be a string?
In the code you could check if the value is a list and if not pack it into a single item list.

````python
if not isinstance(value, list):
  value = [value]
````

Reply:
That's a good suggestion, especially for those, who only plan to notify single mail addresses for each item.